src/Uno.UWP/Helpers/AndroidResourceNameEncoder.cs

```diff
@@ -31,6 +31,16 @@ public static string Encode(string key)
 				key = $"{NumberPrefix}{key}";
 			}
 
+			if (key.EndsWith(".9"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'key.EndsWith' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5653558)

Reply:
IAWTR